frontend/test/__runner__/backend.js

```diff
@@ -195,4 +186,4 @@ function delay(duration) {
   return new Promise((resolve, reject) => setTimeout(resolve, duration));
 }
 
-module.exports = { DEFAULT_DB_KEY, BackendResource, isReady };
```

Comment:
Not only do we NOT use `DEFAULT_DB_KEY` anywhere else in the app but, as it stands currently, it's completely useless even in the context of this file. We'd have to explicitly pass `dbKey` as `undefined` or `null` for its value to fall back to the `DEFAULT_DB_KEY`.

We construct server from [this line](https://github.com/metabase/metabase/pull/24062/files#diff-500eab95775999609b1f7a092ee9ab144e371af3175de7471ed5139891b8913bL9) where we explicitly pass the `dbKey` as an empty string.

https://github.com/metabase/metabase/pull/24062/files#diff-f19428aae1af064bfa29c3e020fc0ca7cccee2af5117f6722f7763f2f750bdc4L20:L22

This logic then kicks in and `dbKey` is resolved as an `{}`.